catalog/opensearch/catalog-opensearch-source/src/main/java/ddf/catalog/source/opensearch/impl/OpenSearchSource.java

```diff
@@ -719,7 +719,15 @@ protected InputTransformer lookupTransformerReference(String namespaceUri)
       Collection<ServiceReference<InputTransformer>> transformerReference =
           bundleContext.getServiceReferences(
               InputTransformer.class, "(schema=" + namespaceUri + ")");
-      return bundleContext.getService(transformerReference.iterator().next());
+      if (transformerReference.isEmpty()) {
+        LOGGER.trace("Failed to find Input Transformer by schema : {}", namespaceUri);
```

Comment:
If the transformer is not found, we just return null below. Does that not cause any unintended side effects? I feel like this change should have a test for the expected behavior in that case.

Reply:
Do we have a good example of how to mock OSGi services in unit tests at this point? Or how to configure and test a loopback source during an itest?

Reply:
I think we already configure loopback for OpenSearch in itests 
https://github.com/codice/ddf/blob/master/distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java#L236

Reply:
@mackncheesiest are there plans to address this comment?